nemo/collections/tts/models/waveglow.py

```diff
@@ -0,0 +1,231 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Dict, Optional
+
+import torch
+from omegaconf import MISSING, DictConfig, OmegaConf, open_dict
+
+from nemo.collections.tts.helpers.helpers import waveglow_log_to_tb_func
+from nemo.collections.tts.modules.waveglow import OperationMode
+from nemo.core.classes import Loss, ModelPT, typecheck
+from nemo.core.neural_types.elements import (
+    AudioSignal,
+    LengthsType,
+    LossType,
+    MelSpectrogramType,
+    NormalDistributionSamplesType,
+    VoidType,
+)
+from nemo.core.neural_types.neural_type import NeuralType
+from nemo.utils import logging
+from nemo.utils.decorators import experimental
+
+
+class WaveGlowLoss(Loss):
```

Comment:
same as for Tacotron2 - could you please move this to nemo/collections/tts/losses/* and give it a docstring

Reply:
Moved